test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -217,6 +219,14 @@ protected void getManagementTestLayout()
             corfuRuntime.getRouter(server).getClient(ManagementClient.class).initiateFailureHandler().get();
         }
 
+        // Waiting for management servers to send the bootstrap sequencer request and be ready
+        // to detect failures to speed up test time.
+        while (!getManagementServer(SERVERS.PORT_0).isSequencerBootstrapped()
```

Comment:
Can we switch this to CompletableFuture.allOf().join() with a timeout (and throw a runtime exception of type assertionFailure if we timeout?)